modules/ROOT/pages/lb-cert-upload.adoc

```diff
@@ -49,8 +49,8 @@ To add a certificate using Runtime Manager:
 image::lb-cert-upload-update.png[Add certificate option in the load balancer settings page]
 . Select *Choose File* to upload the public key and private key files, such as:
 
-** Public key file: `example-com-crt.pem`
-** Private key file: `example-com-private.pem`
+** Public key file (Combined Primary.crt & Intermediate_CA.crt into one file.  File should be converted to PEM extension.): `example-com-crt-public.pem`
+** Private key file (Should be an unencrypted private key -> openssl rsa -in example-com-private.pem -out example-com-private_noencrypt.pem): `example-com-private-noencrypt.pem`
```

Comment:
Why the openssl command? We should just be calling out that it needs to be a passphrase-less private key.

Reply:
That's fine but giving the openssl command will just do it for them rather then trying to figure out ways to make it passphrase-less.